python/shotgun_fields/date_widget.py

```diff
@@ -8,20 +8,45 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-"""
-Widget that represents the value of a date field in Shotgun
-"""
 import datetime
 
-from .date_and_time_widget import DateAndTimeWidget
+from sgtk.platform.qt import QtGui, QtCore
+from .label_base_widget import LabelBaseWidget
+from .shotgun_field_meta import ShotgunFieldMeta
 
 
-class DateWidget(DateAndTimeWidget):
+class DateWidget(LabelBaseWidget):
     """
-    Inherited from a :class:`~DateAndTimeWidget`, this class is able to
-    display a date field value as returned by the Shotgun API.
+    Display a ``date`` field value as returned by the Shotgun API.
     """
-    _FIELD_TYPE = "date"
+    __metaclass__ = ShotgunFieldMeta
+    _DISPLAY_TYPE = "date"
+
+    def _create_human_readable_date(self, date):
```

Comment:
FYI, this logic has been moved into shotgun_globals (I think that's where I put it...) in newer code.